problems/removeNumbersAtOddIndices.js

```diff
@@ -2,8 +2,15 @@
 * @param {number[]} arr - The input array
 * @returns {number[]} - An array removing all elements initially appearing at an odd index
 */
-function removeNumbersAtOddIndices() {
-
+const removeNumbersAtOddIndices = (arr) => {
+    let output = []
+    for(let i = 0; i < arr.length; i++){
+        if(i % 2 === 1){
+            output.push(arr[i])
+            arr = output
```

Comment:
Not so sure about this line... we don't want to reassign our array to be our output with each odd index.